src/components/Footer/Footer.scss

```diff
@@ -0,0 +1,9 @@
+.footer {
```

Comment:
do you have any instructions on the CSS for this page?
The page works but it looks a little choppy and slightly out of place form time to time. The footer is bulging out and quite thick taking up half of the page.

 Have a look at some real hotel pages for some inspiration on what would look good and cohesive. What do their footers look like? What makes them good?  Here are some examples: 
 https://www.oneandonlyresorts.com/cape-town
 https://www.samsaraubud.com/